components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/OperationPoliciesApiServiceImpl.java

```diff
@@ -202,29 +220,72 @@ public Response deleteCommonOperationPolicyByPolicyId(String operationPolicyId,
      * @param messageContext message context
      * @return A list of operation policies available for the API
      */
-    @Override
-    public Response getAllCommonOperationPolicies(Integer limit, Integer offset, String query,
-                                                  MessageContext messageContext) throws APIManagementException {
+    @Override public Response getAllCommonOperationPolicies(Integer limit, Integer offset, String query,MessageContext messageContext) throws APIManagementException {
```

Comment:
Let's revert this change to have Override in one line and the method in the next line. (This is an issue in the auto formatter)

Reply:
Fixed by [d413768](https://github.com/wso2/carbon-apimgt/pull/11325/commits/d41376893b76bc5aaafe2e28c65b6da4dd2d5642)